en/developer/back-end.md

```diff
@@ -0,0 +1,17 @@
+# Tips for back-end developers
+
+## Use PHP’s built-in web server
+
+[PHP’s built-in web server](https://www.php.net/manual/en/features.commandline.webserver.php) allows you to develop without installing a web server like Nginx or Apache.
+
+To install PHP dependencies, configure Wallabag and start the built-in server, do:
+
+```
+./scripts/dev.sh
+```
+
+If you only want to start the built-in server:
+
+```
+php bin/console server:run
```

Comment:
This won't be available anymore when we upgrade to Symfony 4.

Reply:
Ok. Anyway, I noticed that you need to build the `dev` version of the assets to be able to use the built-in server, so I’ll change my doc to `make dev`